src/aiida/cmdline/params/options/main.py

```diff
@@ -92,6 +92,7 @@
     'REPOSITORY_PATH',
     'SCHEDULER',
     'SILENT',
+    'SORT_OUTPUT',
```

Comment:
This can now be removed right?
```suggestion
    'SORT_OUTPUT',
```